client/src/modules/stock/entry/modals/templates/transfer.row.tmpl.html

```diff
@@ -0,0 +1,9 @@
+<div 
+  ng-hide="row.entity.countedReceived > 0 && !grid.appScope.filterReceived"
```

Comment:
Why do the filter on a row template?  Why not just filter the data in the controller?

This seems to me overly complicated.